### PR TITLE
Feature/group 3 spring backend

### PR DESCRIPTION
**Pull request by nany-d**

### CODE REVIEW

File: sqlScript.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
+insert into `vehicle_registration`
```
Comment (SJKnapp) [inline sqlScript.txt]: should .sql not .txt

File: Group3-Backend-Spring/src/main/resources/application-prod.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
+server.port=8080
+
+jwt.secret = javainuse
+jwt.expirationDateInMs=180000
+jwt.refreshExpirationDateInMs=9000000
+
+spring.datasource.url=${dburl}
```
Comment (SJKnapp) [inline Group3-Backend-Spring/src/main/resources/application-prod.properties]: Sorry i know i was the one who gave you these but could they change dburl to DBUrl and dbpassword to DBPassword

**Review by SJKnapp — COMMENTED** [review]
left a few comments

File: Group3-Backend-Spring/src/main/resources/application-prod.properties [hunk 3]
```diff
@@ -0,0 +1,11 @@
+server.port=8080
+
+jwt.secret = javainuse
```
Comment (RyanCD1) [inline Group3-Backend-Spring/src/main/resources/application-prod.properties]: hide in jenkins env variable?

Comment (SJKnapp) [reply Group3-Backend-Spring/src/main/resources/application-prod.properties]: Call it JWTSecret

**Review by SJKnapp — APPROVED** [review]

**Review by RyanCD1 — APPROVED** [review]

**Pull request merged**